### DISCUSSION TITLE
Telegram API not found

### DISCUSSION THREAD

**Discussion by IgorZanellaDev** [jef/streetmerchant#1333] [Help] [answered]
Hi, I downloaded and launched the bot from docker, but when I launch the thing to try notification on Telegram, it says this.

`[9:36:45 PM] info :: ✔ telegram message sent
(node:17) UnhandledPromiseRejectionWarning: AxiosError: Telegram API - 404 Not Found
    at TelegramClient.<anonymous> (/app/node_modules/messaging-api-telegram/dist/TelegramClient.js:67:27)
    at Generator.throw (<anonymous>)
    at rejected (/app/node_modules/messaging-api-telegram/dist/TelegramClient.js:6:65)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:17) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:17) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
`

Am I wrong or is there a problem on telegram API? I used the next docker command

`docker run --cap-add=SYS_ADMIN   -it --rm --env-file ./.env   ghcr.io/jef/streetmerchant:nightly test:notification:production`.

Thank you.

**Comment by jef** [comment, marked as answer]
Interesting. I'll see if I can duplicate.

**Reply by jef** [reply]
Found the issue. Please remove the environment variable if it isn't in use. Docker has problems reading in environment variables with quotes.
